crates/sargon-uniffi/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/subintent_manifest.rs

```diff
@@ -45,13 +45,6 @@ pub fn subintent_manifest_string(manifest: &SubintentManifest) -> String {
     manifest.into_internal().manifest_string()
 }
 
-#[uniffi::export]
-pub fn subintent_manifest_summary(
-    manifest: &SubintentManifest,
-) -> Option<ManifestSummary> {
-    manifest.into_internal().summary().map(|s| s.into())
-}
```

Comment:
I believe we are gonna need to add the following:
```Rust
#[uniffi::export]
pub fn subintent_manifest_instructions_string(manifest: &SubintentManifest) -> String {
    manifest.into_internal().instructions_string()
}
```

In order to expose the raw instructions ([design](https://zpl.io/QM0yGJW))

Reply:
We would be using the above manifest_string